basic/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Basic Portfolio" />
+    <title>Basic Portfolio</title>
```

Comment:
Please follow the title syntax in the CONTRIBUTING.md

Reply:
Done guru